app/scripts/modules/core/utils/renderIfFeature.directive.ts

```diff
@@ -0,0 +1,34 @@
+import {module} from 'angular';
+import {DirectiveFactory} from './tsDecorators/directiveFactoryDecorator';
+
+class RenderIfFeatureDirectiveController {
+  public $attrs: ng.IAttributes;
+  public $element: JQuery;
+
+  constructor(private settings: any) {}
+
+  public initialize(): void {
+    let feature: string = this.$attrs['renderIfFeature'];
```

Comment:
We just updated to TS 2.2, which lets you specify this as `this.$attrs.renderIfFeature` (I think) - can you try that and see if the compiler complains?